wiz-outpost-lite/templates/deployment.yaml

```diff
@@ -15,10 +15,12 @@ spec:
   template:
     metadata:
       annotations:
-      {{- with .Values.podAnnotations }}
+        {{- with .Values.podAnnotations }}
         {{- toYaml . | nindent 8 }}
-      {{- end }}
+        {{- end }}
+        {{- if .Values.podAnnotationsApparmor.enabled }}
```

Comment:
I would call this `apparmorSecurityProfile`, make it a string, and default for non remediations module to 'unconfined'